test/middleware/request_validation_open_api_3_test.rb

```diff
@@ -50,12 +50,24 @@ def app
       [200, {}, []]
     }
 
-    @app = new_rack_app_with_lambda(check_parameter, schema: open_api_3_schema, coerce_date_times: true)
+    @app = new_rack_app_with_lambda(check_parameter,
+                                    schema: open_api_3_schema,
+                                    coerce_date_times: true,
+                                    allow_get_body: true)
 
     get "/get_body_test", { no_problem: true }, { input: params.to_json }
     assert_equal 200, last_response.status
   end
 
+  it "get error given a datetime and with coerce_date_times enabled on GET endpoint with request body" do
+    params = { "datetime_string" => "2016-04-01T16:00:00.000+09:00" }
+
+    @app = new_rack_app(schema: open_api_3_schema, allow_get_body: false)
+
+    get "/get_body_test", { no_problem: true }, { input: params.to_json }
+    assert_equal 400, last_response.status
+  end
```

Comment:
Can we try to write a simpler test case for this one that doesn't mention `coerce_date_times` (or any other options in any way)? Ideally 

```
it "doesn't respect values sent in `GET` request body when `allow_get_body` is false" do
```